### DISCUSSION TITLE
Is version 2.5.1 safe?

### DISCUSSION THREAD

**Discussion by y-takebe** [uilibrary/Octavia-admin-Pro-Issues#14] [Q&A] [answered]
I recently received version 2.5.1 of Octavia Admin Pro from The SendOwl.

The correction points are as follows,

> The seller has added the following message for this update:
>
> This update includes:
> --Chat Ui Issue fix

However, the code difference from 2.5.0 includes the following Google Analytics Tracking Code:

```
--- a/octavia-admin-pro-2.5.0/Octavia-2.5.0_Octavia-starter-kit/Octavia-2.5.0/public/index.html
+++ b/octavia-admin-pro-2.5.1/Octavia-2.5.1/public/index.html
@@ -10,6 +10,16 @@
             sizes="180x180"
             href="<%= BASE_URL %>apple-touch-icon.png"
         />
+           <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-103569775-1"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'UA-103569775-1');
+</script>
+
```

**Is version 2.5.1 safe?**

The usage status of the purchaser and the behavior of the user after the purchaser has incorporated it into the product may be transmitted to a third party (the owner of the tracking code UA-103569775-1).

**Comment by naime-hossain** [comment, marked as answer]
> I recently received version 2.5.1 of Octavia Admin Pro from The SendOwl.
> 
> The correction points are as follows,
> 
> > The seller has added the following message for this update:
> >
> > This update includes:
> > --Chat Ui Issue fix
> 
> However, the code difference from 2.5.0 includes the following Google Analytics Tracking Code:
> 
> ```
> --- a/octavia-admin-pro-2.5.0/Octavia-2.5.0_Octavia-starter-kit/Octavia-2.5.0/public/index.html
> +++ b/octavia-admin-pro-2.5.1/Octavia-2.5.1/public/index.html
> @@ -10,6 +10,16 @@
>              sizes="180x180"
>              href="<%= BASE_URL %>apple-touch-icon.png"
>          />
> +           
> +<script async src="https://www.googletagmanager.com/gtag/js?id=UA-103569775-1"></script>
> +<script>
> +  window.dataLayer = window.dataLayer || [];
> +  function gtag(){dataLayer.push(arguments);}
> +  gtag('js', new Date());
> +
> +  gtag('config', 'UA-103569775-1');
> +</script>
> +
> ```
> 
> **Is version 2.5.1 safe?**
> 
> The usage status of the purchaser and the behavior of the user after the purchaser has incorporated it into the product may be transmitted to a third party (the owner of the tracking code UA-103569775-1).

Its safe, you can easily remove it 🙂 Sorry i forget to remove the code, i will remove it and update soon. Thank you

**Reply by y-takebe** [reply]
@naime-hossain 
I got it. thanks!

**Reply by naime-hossain** [reply]
> @naime-hossain
> I got it. thanks!

Welcome
